### PR TITLE
Remove unused companion objects on Response type

### PR DESCRIPTION
**Pull request by fredells**
This is part 3 of generated code cleanup to improve build times.

It seems these companion objects are completely unused and negatively affect build times.

### CODE REVIEW

File: Templates/Kotlin/ResponseType.stencil [hunk 1]
```diff
@@ -29,15 +29,13 @@ data class {{ name }}(
         {% endfor %}
     )
 
+    {% if fragmentSelections %}
     companion object {
-        const val typeName = "{{ graphQLName }}"
```
Comment (jaredh) [inline Templates/Kotlin/ResponseType.stencil:33]: Is it useful to keep this from an API POV? 🤔 Probably not if it means keeping companion objects around on every response.

File: Templates/Kotlin/ResponseType.stencil [hunk 2]
```diff
@@ -29,15 +29,13 @@ data class {{ name }}(
...
-
-        {% if fragmentSelections %}
         fun getSelections(operationVariables: Map<String, String>): List<Selection> {
```
Comment (jaredh) [inline Templates/Kotlin/ResponseType.stencil:34]: While in the area, I'd consider updating to a `selections` property accessor.

Comment (fredells) [reply Templates/Kotlin/ResponseType.stencil:34]: dont think we can since we need to pass operationVariables

Comment (jaredh) [reply Templates/Kotlin/ResponseType.stencil:34]: You're totally right. I just ghosted over that when I saw get prefix. 👍

**Pull request merged**